src/nnsight/models/UnifiedTransformer.py

```diff
@@ -28,13 +28,13 @@ class UnifiedTransformer(LanguageModel):
     """
 
     def __init__(
-        self, model: str, *args, processing: bool = True, **kwargs
+        self, model: str, *args, processing: bool = True, tokenizer: Optional[PreTrainedTokenizer] = None, **kwargs
```

Comment:
You need to import Optional. After that I'll merge it in!

Reply:
done!